Gemfile

```diff
@@ -2,8 +2,8 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'rake'
-gem 'rake-compiler'
-gem 'rake-compiler-dock'
+gem 'rake-compiler', `~> 1.1.7`
```

Comment:
Actually, the CI failures were due to the apostrophes on this line. 😄

Reply:
fffuuu thank you for spotting this

Reply:
Fixed.